### DISCUSSION TITLE
Vercel's DNS servers (MX value)

### DISCUSSION THREAD

**Discussion by Vichp** [vercel/community#3756] [Help] [answered]
### Summary

Hello, 

What I would like to achieve is the following: I have aquired a domain name, and I would like to add a website and a mail server for this domain.  I have: 1 Domain name, 1 private Ubuntu server with IP,  Vercel.

For the website, I am hosting it on Vercel, and linked the DNS servers to use Vercel's. 

As for the email, I would like to use my own private server on Digital Ocean, basically an Ubuntu box that I SSH into. I could add a MX entry on Vercel and point it to my private server's IP address, however as per the MX specification - the MX has to be a domain name and not an IP address. 

Does anyone know if Vercel allows for the MX address to be an IP ? Or is there a better solution to my issue? 

Thanks in advance.

### Steps to Reproduce

-

**Comment by amyegan** [comment, marked as answer]
Hey @Vichp. If the domain is registered with Vercel, or it uses Vercel's nameservers, then you can manage DNS records from the [Domains tab in your Vercel Dashboard](https://vercel.com/dashboard/domains). More info is available in the [Managing DNS Records](https://vercel.com/docs/projects/domains/managing-dns-records) doc and the "[How do I send and receive emails with my Vercel purchased domain?](https://vercel.com/guides/using-email-with-your-vercel-domain)" guide.

If you used A and CNAME records to point the domain to your project, then you'll need to manage the DNS records for your email connection in the same place. Your domain registrar should be able to help with this if needed.
